### ISSUE TITLE
Body25B caffemodel download

### ISSUE THREAD

**Issue by Witcet** [perfanalytics/pose2sim#27]
Hello,

Thank you for your excellent work. Currently, I'm using the Body25 model for human pose detection, and it's performing really well. However, I noticed in the pose2sim documentation that using the Body25B model might yield even better results. Therefore, I would like to try using the Body25B model. Unfortunately, I couldn't find the Body25B pose_iter_XXXXXX.caffemodel file in the openpose git repository. I'm wondering if it is necessary to train this model myself with Caffe or if there are pre-trained model files available similar to the ones provided for the Body25 model. If there are pre-trained Body25B models available, could you kindly guide me on how to download them?

Thank you for your assistance.

**Comment by davidpagnon** [comment]
Hello,

You're welcome!
You don't have to train anything, you can just go [there](https://docs.google.com/presentation/d/1dWVPyqekEDOVnxZB9u0h7vL4ofg-Qe8qThCaELKFUHw/edit?usp=sharing) and download the model and the prototxt file into your `openpose/models/pose/body_25b` directory.

Direct links:
- Model [there](http://posefs1.perception.cs.cmu.edu/OpenPose/models/pose/1_25BBkg/body_25b/pose_iter_XXXXXX.caffemodel)
- Prototxt file [there](https://raw.githubusercontent.com/CMU-Perceptual-Computing-Lab/openpose_train/master/experimental_models/1_25BBkg/body_25b/pose_deploy.prototxt)

**Comment by Witcet** [comment]
Thank you for your help！
I finally figured out why I couldn't download the pose_iter_XXXXXX.caffemodel from the official link before. It turned out to be due to the default security policy of Google Chrome browser. When I directly clicked on the download link, the download page would just flicker and not start the download. All I needed to do was copy the download link to the Google Chrome address bar and press enter，then I can download it successfully. Haha, I made a silly mistake.
